ui/react-components/components/DatePickerCarbon/DatePickerCarbon.jsx

```diff
@@ -5,12 +5,16 @@ import moment from "moment";
 import Title from "../Title/Title.jsx";
 const DatePickerCarbon = props => {
 
-    const {onChange, value, title, minDate, testId, width, isDisabled, isRequired} = props;
+    const {onChange, value, title, minDate, testId, width, isDisabled, isRequired, showWarning, intl} = props;
     let defaultTime = value;
     if( value && value instanceof moment){
         defaultTime = value.format("MM/DD/YYYY");
     }
     let titleText=  title && <Title text={title} isRequired={isRequired}/>
+    const warningText = intl.formatMessage({
+        id: 'PUBLIC_HOLIDAY_WARNING',
+        defaultMessage: 'Date selected is a Public Holiday',
```

Comment:
@umair-fayaz can you check with BA's this is the message we want to see on UI

Reply:
Already confirmed that Bindu